### PR TITLE
Add federatedproperties modal

### PR DESCRIPTION
**Pull request by toban**

### CODE REVIEW

**Comment by addshore** [comment]
Preview URLs:
https://06f8fa2122-mock--wbstack-ui.netlify.app
https://06f8fa2122-prod--wbstack-ui.netlify.app

File: src/components/Pages/ManageWiki/Features/FederatedProperties.vue [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            <v-spacer></v-spacer>
+            <v-btn icon dark @click="hideModal">
+              <v-icon>close</v-icon>
+            </v-btn>
+          </v-toolbar>
+        <br />
+        <v-card-text>If you turn this on, and confirm the warning, any existing properties on your wikibase will no longer be accessible!</v-card-text>
+        <v-card-text>
+          <v-text-field
+            v-model="userConfirmationInput"
```
Comment (jakobw) [inline src/components/Pages/ManageWiki/Features/FederatedProperties.vue:26]: Getting some console spam for `Property or method "userConfirmationInput" is not defined on the instance but referenced during render.`. This isn't used anyway so we can just delete it.

```suggestion
```

File: src/components/Pages/ManageWiki/Features/FederatedProperties.vue [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    }
+  },
+  methods: {
+    enableFeature () {
+      const wiki = this.wikiId
+      const setting = 'wikibaseFedPropsEnable'
+      const value = 1
+
+      this.$store.dispatch('updateSetting', { wiki, setting, value })
+        .then(() => this.$router.push('/dashboard'))
```
Comment (jakobw) [inline src/components/Pages/ManageWiki/Features/FederatedProperties.vue:64]: Being sent back to the dashboard caught me by surprise. I expected the modal to just disappear and to get a confirmation somewhere on the same page. Should be fine for now but maybe that's something to poke UX about.

File: src/components/Pages/ManageWiki/Tabs/ManageFeatures.vue [hunk 3]
```diff
@@ -1,13 +1,22 @@
...
 <script>
+import FederatedProperties from '../Features/FederatedProperties.vue'
+
 export default {
   name: 'ManageFeatures',
-  components: {
+  components: { FederatedProperties },
+  data () {
+    return {
+      wikiId: 0
```
Comment (jakobw) [inline src/components/Pages/ManageWiki/Tabs/ManageFeatures.vue:15]: I feel like `null` is more appropriate as an initial value. `0` might be an actual wiki id.

```suggestion
      wikiId: null
```

File: src/components/Pages/ManageWiki/Tabs/ManageFeatures.vue [hunk 4]
```diff
@@ -1,13 +1,22 @@
 <template>
   <v-content>
-      Feature Settings
+    <FederatedProperties :wikiId="this.wikiId" />
```
Comment (jakobw) [inline src/components/Pages/ManageWiki/Tabs/ManageFeatures.vue:3]: I'm amazed that it works with `this.`. Pretty sure it shouldn't be there, though! :D
```suggestion
    <FederatedProperties :wikiId="wikiId" />
```

**Review by jakobw — CHANGES_REQUESTED** [review]
Looks great, left some minor comments here and there.

File: src/components/Pages/ManageWiki/Tabs/ManageFeatures.vue (hunk 4 above)
Comment (toban) [reply src/components/Pages/ManageWiki/Tabs/ManageFeatures.vue]: done!

File: src/components/Pages/ManageWiki/Tabs/ManageFeatures.vue (hunk 3 above)
Comment (toban) [reply src/components/Pages/ManageWiki/Tabs/ManageFeatures.vue]: done!

File: src/components/Pages/ManageWiki/Features/FederatedProperties.vue (hunk 2 above)
Comment (toban) [reply src/components/Pages/ManageWiki/Features/FederatedProperties.vue]: done!

File: src/components/Pages/ManageWiki/Features/FederatedProperties.vue (hunk 1 above)
Comment (toban) [reply src/components/Pages/ManageWiki/Features/FederatedProperties.vue]: done!

**Comment by addshore** [comment]
Looks like a good state to merge, and we can do more tommorrow

**Pull request merged**